cnxarchive/sql/get-in-book-search.sql

```diff
@@ -0,0 +1,40 @@
+-- ###
+-- Copyright (c) 2013, Rice University
+-- This software is subject to the provisions of the GNU Affero General
+-- Public License version 3 (AGPLv3).
+-- See LICENCE.txt for details.
+-- ###
+
+-- arguments: uuid:string, major_version:int, minor_version:int, search_term:string
+WITH RECURSIVE t(node, title, path,value, depth, corder) AS (
+SELECT nodeid, title, ARRAY[nodeid], documentid, 1, ARRAY[childorder]
+FROM 
+  trees tr, 
+  modules m
+WHERE 
+  m.uuid::text = %(uuid)s AND
+  m.major_version = %(major_version)i AND  m.minor_version = %(minor_version)i AND
```

Comment:
The only supported format specifier is 's', so this should be %(major_version)s Psycopg does the right thing if you pass it an integer: